src/scripts.js

```diff
@@ -246,13 +246,13 @@ fetchData('users/userData')
       userIDs.forEach(userID => {
         let userName = findUserName(Number(userID));
         list += `<li class="friends_li">
-               <p class="friends--steps"><b>${userName}</b>:</p>
```

Comment:
Removing this and styling the name was a good idea.  I think it makes more sense from an accessibility and screen reader standpoint too.